README.md

```diff
@@ -1,17 +1,95 @@
-# account-graph
+# Account graph
 
 `account-graph` is a smart contract in Move language on Sui network.
-It captures relationships between accounts using a directed graph, where nodes are addresses and edges are pairs of addresses.
+It captures relationships between accounts using a _directed graph_, where nodes are Sui addresses and edges are relationships between a pair of Sui addresses.
 It also provides the ability to store properties of nodes and edges along with the graph.
 All graph operations are authorized by the source account.
```

Comment:
It would be good to clarify the term "source account" just a little to avoid confusion, e.g. 
"... the source account, and not the addresses being linked to. Addressed being linked to via an account graph have no control over the existence or properties of the link."

As opposed to the source of the graph instance, which also may impose rules.

Reply:
I meant for this to reference line 6